sdk/metric/sdk.go

```diff
@@ -305,11 +321,17 @@ func (s *syncInstrument) RecordOne(ctx context.Context, number api.Number, kvs [
 // processor will call Collect() when it receives a request to scrape
 // current metric values.  A push-based processor should configure its
 // own periodic collection.
-func NewAccumulator(processor export.Processor, resource *resource.Resource) *Accumulator {
+func NewAccumulator(processor export.Processor, resource *resource.Resource, opts ...Option) *Accumulator {
```

Comment:
See the comment above. I'd be happier to leave this a three-argument function until we have more than one optional setting.